packages/contractkit/src/governance/proposals.ts

```diff
@@ -0,0 +1,96 @@
+import { concurrentMap } from '@celo/utils/lib/async'
+import { keccak256 } from 'ethereumjs-util'
+import Contract from 'web3/eth/contract'
+import { Transaction, TransactionObject } from 'web3/eth/types'
+import { AllContracts, CeloContract } from '../base'
+import { obtainKitContractDetails } from '../explorer/base'
+import { BlockExplorer } from '../explorer/block-explorer'
+import { ContractKit } from '../kit'
+import { CeloTransactionObject, valueToString } from '../wrappers/BaseWrapper'
+import { Proposal, ProposalTransaction } from '../wrappers/Governance'
+
+export interface ProposalTransactionJSON {
+  contract: CeloContract
+  function: string
+  args: any[]
+  value: string
+}
+
+export class ProposalUtility extends Proposal {
```

Comment:
seems weird for an utility class to extend from a Value class... i think composition work better